localization/english/religion/wc_religion_l_english.yml

```diff
@@ -192,6 +192,12 @@
  scarlet_light_adherent:0 "Scarlet Follower"
  scarlet_light_adherent_plural:0 "Scarlet Followers"
  scarlet_light_desc:0 "A heretical, militant offshoot of the Church of the Holy Light. They are intolerant of all other faiths, regardless of alliance or affiliation. They believe that any and all unholy creatures must be destroyed by a cleansing flame."
+
+ kirin_tor:0 "Kirin Torism"
+ kirin_tor_adj:0 "Kirin Toran"
+ kirin_tor_adherent:0 "Kirin Toran"
+ kirin_tor_adherent_plural:0 "Kirin Torans"
+ kirin_tor_desc:0 "Located in the magical city of Dalaran, the Kirin Tor hope to instil a sense of respect and wonder for the arcane arts into the new generation of mage hopefuls, all while delving deeper into the secrets of Magic itself. While some of these mages worship magic as a whole, those from Lordaeron still venerate the Holy Light, albeit with some arcane twists."
```

Comment:
Another comment, so I wouldn't mention where the Kirin Tor located because it may change during the game.
You could mention it's originated from Dalaran. Speaking about the past is welcome.